### ISSUE TITLE
IOException

### ISSUE THREAD

**Issue by mamu911** [maximilianvoss/spotify-toniebox-sync#19]
I try to run your spotify-tonie-sync... but I do not get it work, after some installation trouble I am now able to run it. I try booth single and daemon... here are the log:

I hope someone can help me out... (I just installed the spotify-toniebox-sync and NOT the toniebox-api, I guess this is inclued in the spotify sync?)

Thanks, Matthias 

10 Oct 2020 01:45:59,738 - ERROR:rocks.voss.spotifytonieboxsync.Application:main:71: IOException
com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "errors" (class rocks.voss.toniebox.beans.toniebox.JWTToken), not marked as ignorable (one known property: "jwt"])
 at [Source: N/A; line: -1, column: -1] (through reference chain: rocks.voss.toniebox.beans.toniebox.JWTToken["errors"])
	at com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException.from(UnrecognizedPropertyException.java:79)
	at com.fasterxml.jackson.databind.DeserializationContext.reportUnknownProperty(DeserializationContext.java:555)
	at com.fasterxml.jackson.databind.deser.std.StdDeserializer.handleUnknownProperty(StdDeserializer.java:708)
	at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownProperty(BeanDeserializerBase.java:1160)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserializeFromObject(BeanDeserializer.java:315)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:121)
	at com.fasterxml.jackson.databind.ObjectMapper._readValue(ObjectMapper.java:2860)
	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:1569)
	at com.fasterxml.jackson.databind.ObjectMapper.treeToValue(ObjectMapper.java:1902)
	at rocks.voss.toniebox.beans.Transformer.createBean(Transformer.java:23)
	at rocks.voss.toniebox.beans.Transformer.createBean(Transformer.java:16)
	at rocks.voss.toniebox.RequestHandler.executeRequest(RequestHandler.java:149)
	at rocks.voss.toniebox.RequestHandler.executePostRequest(RequestHandler.java:115)
	at rocks.voss.toniebox.RequestHandler.Login(RequestHandler.java:47)
	at rocks.voss.toniebox.TonieHandler.login(TonieHandler.java:45)
	at rocks.voss.spotifytonieboxsync.Application.main(Application.java:61)

**Comment by availle** [comment]
Same here.

**Comment by paschmal** [comment]
Seems like Toniecloud API changed again. It tried to fix this by adding a new field `errors` into JWTToken.java (see my fork: [https://github.com/paschmal/toniebox-api/commit/759fb451b1b3166657c756c5739b9f60645fdc7c](https://github.com/paschmal/toniebox-api/commit/759fb451b1b3166657c756c5739b9f60645fdc7c)).
But unfortunately I got another error:

```
Exception in thread "main" com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "detail" (class rocks.voss.toniebox.beans.toniebox.Me), not marked as ignorable (15 known properties: , "lastName", "notificationCount", "requiresVerificationToUpload", "sex", "verified", "acceptedTermsOfUse", "profileImage", "isVerified", "authCode", "tracking", "isEduUser", "locale" [truncated]])
 at [Source: N/A; line: -1, column: -1] (through reference chain: rocks.voss.toniebox.beans.toniebox.Me["detail"])
        at com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException.from(UnrecognizedPropertyException.java:79)
        at com.fasterxml.jackson.databind.DeserializationContext.reportUnknownProperty(DeserializationContext.java:555)
        at com.fasterxml.jackson.databind.deser.std.StdDeserializer.handleUnknownProperty(StdDeserializer.java:708)
        at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownProperty(BeanDeserializerBase.java:1160)
        at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserializeFromObject(BeanDeserializer.java:315)
        at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:121)
        at com.fasterxml.jackson.databind.ObjectMapper._readValue(ObjectMapper.java:2860)
        at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:1569)
        at com.fasterxml.jackson.databind.ObjectMapper.treeToValue(ObjectMapper.java:1902)
        at rocks.voss.toniebox.beans.Transformer.createBean(Transformer.java:23)
        at rocks.voss.toniebox.beans.Transformer.createBean(Transformer.java:16)
        at rocks.voss.toniebox.RequestHandler.executeRequest(RequestHandler.java:168)
        at rocks.voss.toniebox.RequestHandler.executeGetRequest(RequestHandler.java:120)
        at rocks.voss.toniebox.RequestHandler.getMe(RequestHandler.java:61)
        at rocks.voss.toniebox.TonieHandler.getMe(TonieHandler.java:74)
        at rocks.voss.toniebox.example.Application.main(Application.java:24)
```

But the field `detail` is not documented in the Toniecloud API Documentation [https://api.tonie.cloud/v2/doc/](https://api.tonie.cloud/v2/doc/). Does anybody has an idea how to fix this?

**Comment by maximilianvoss** [comment]
Was an outdated toniebox-api version. Fixed for v2.0
